### ISSUE TITLE
remove support for embed

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-pipeline#1060]
the embed feature using ESI includes is very brittle and causes problems often times. embedding is better done clientside.

**Comment by kptdobe** [comment]
For the record: theblog project use "ESI" embeds. Removing support would mean you potentially break a lot of existing blog posts. Migrating the content is not an option (more than 20k md and docx pages to check and update with the corresponding block).
Only option would be to write some logic to detect those embeds and transform them into block... client side.

**Comment by tripodsan** [comment]
> Only option would be to write some logic to detect those embeds and transform them into block... client side.

yes. this will be _the_ option. so we need to prepare the breaking branch accordingly.
